backend/src/modules/access-request/repositories/access-request.repository.ts

```diff
@@ -74,6 +75,33 @@ export class AccessRequestRepository
     );
   }
 
+  async getManyForDownload(
+    findOptions: FindAccessRequestOptions,
+  ): Promise<IAccessRequestDownload[]> {
+    const accessRequests = await this.findMany({
+      ...findOptions,
+      limit: 0,
+      page: 0,
+    });
+
+    return accessRequests.items.map((accessRequest): IAccessRequestDownload => {
```

Comment:
Maparea asta se va intampla doar odata, doar pe usecase-ul de download.

Propun sa nu adaugam logica de download cu getManyForDownload si in repo si in facade, ci sa facem maparea direct in usecase. Practic noi nu facem download-ul de aici, doar luam niste date.